WalletWasabi/WabiSabi/Backend/Banning/CoinVerifier.cs

```diff
@@ -245,13 +242,16 @@ public bool TryScheduleVerification(Coin coin, [NotNullWhen(true)] out CoinVerif
 					if (delay > TimeSpan.Zero)
 					{
 						// We only abort and throw from the delay. If the API request already started, we will go with it.
-						using CancellationTokenSource delayCts = CancellationTokenSource.CreateLinkedTokenSource(linkedCts.Token, item.Token);
+						using CancellationTokenSource delayCts = CancellationTokenSource.CreateLinkedTokenSource(verificationCancellationToken, item.Token);
 						await Task.Delay(delay, delayCts.Token).ConfigureAwait(false);
 					}
 
 					// This is the last chance to abort with abortCts.
 					item.ThrowIfCancellationRequested();
 
+					using CancellationTokenSource requestTimeoutCts = new(ApiRequestTimeout);
+					using CancellationTokenSource linkedCts = CancellationTokenSource.CreateLinkedTokenSource(verificationCancellationToken, item.Token, requestTimeoutCts.Token);
+
```

Comment:
This code looks correct but I would suggest to use this: https://github.com/zkSNACKs/WalletWasabi/blob/master/WalletWasabi/Extensions/CancellationTokenExtensions.cs

Basically, instead of having the same kind of block repeated again and again, you can replace it with this:
`c#
using var linkedCts = verificationCancellationToken.CreateLinkedTokenSourceWithTimeout(ApiRequestTimeout);
`

Reply:
I didn't know this extension method existed. Ty, I will keep this in mind for the future.

But here I need to link 3 different cancellation token together, the extension method doesn't help to make the code prettier in this method IMO.